packages/components/src/box-control/axial-input-controls.tsx

```diff
@@ -129,25 +145,45 @@ export default function AxialInputControls( {
 						? selectedUnits.top
 						: selectedUnits.left;
 				return (
-					<UnitControl
-						{ ...props }
-						isFirst={ first === side }
-						isLast={ last === side }
-						isOnly={ only === side }
-						value={ [
-							parsedQuantity,
-							selectedUnit ?? parsedUnit,
-						].join( '' ) }
-						onChange={ createHandleOnChange( side ) }
-						onUnitChange={ createHandleOnUnitChange( side ) }
-						onFocus={ createHandleOnFocus( side ) }
-						onHoverOn={ createHandleOnHoverOn( side ) }
-						onHoverOff={ createHandleOnHoverOff( side ) }
-						label={ LABELS[ side ] }
-						key={ side }
-					/>
+					<HStack key={ `box-control-${ side }` }>
+						<FlexedBoxControlIcon side={ side } sides={ sides } />
+						<UnitControl
+							{ ...props }
+							isFirst={ first === side }
+							isLast={ last === side }
+							isOnly={ only === side }
+							value={ [
+								parsedQuantity,
+								selectedUnit ?? parsedUnit,
+							].join( '' ) }
+							onChange={ createHandleOnChange( side ) }
+							onUnitChange={ createHandleOnUnitChange( side ) }
+							onFocus={ createHandleOnFocus( side ) }
+							onHoverOn={ createHandleOnHoverOn( side ) }
+							onHoverOff={ createHandleOnHoverOff( side ) }
+							label={ LABELS[ side ] }
+							key={ side }
+						/>
+						<FlexedRangeControl
+							__nextHasNoMarginBottom
+							hideLabelFromVision
+							initialPosition={ 0 }
+							label={ LABELS[ side ] }
+							onChange={ ( newValue ) => {
+								createSliderOnChange?.(
```

Comment:
The `createSliderOnChange` function should be always defined, no need for optional chaining

```suggestion
								createSliderOnChange(
```